pkg/api/v1/auxpb/aux_service.proto

```diff
@@ -15,6 +31,15 @@ message ValidateOauthRequest {
 message ValidateOauthResponse {}
 
 service DSSAuxService {
+  // /dss/version
+  //
+  // Queries the version of the DSS.
+  rpc GetVersion(GetVersionRequest) returns (GetVersionResponse) {
+    option (google.api.http) = {
+      get: "/v1/dss/version"
```

Comment:
I see the issue, but would like to keep some sort of API versioning in place. Any objection to `/dss/v1/aux` or `/dss/v1/self`?

Reply:
Alternatively, `/self/v1` or `/aux/v1`.

Reply:
Eventually, I think we'll want RID stuff to be located at /rid (so, example.com/rid/dss/v1/identification_service_areas for instance) and SCD stuff to be located at /scd (so, example.com/scd/v1/dss/operation_references for instance).  With that naming convention, I would put this at /v1/version in the proto and then prepend "aux" in whatever way we would prepend "rid" and "scd" in the future.  Second best would be to put /aux/v1/version directly in the proto (with the implication that we would pre-process the raw APIs for RID & SCD to add appropriate prefixes in the future).

Reply:
Hmmm, I'm not following. I would have expected that we would aim for something like `/dss/{rid,scd,aux}/v*/...` going forward, thus supporting with a common prefix:
```
/dss/rid/v1/...
/dss/rid/v2/...
/dss/scd/v1/...
/dss/aux/v1/...
```

Reply:
The standard APIs specify particular prefixes -- I'm not sure that's a great approach, but the RID standard writes, [for instance](https://github.com/uastech/standards/blob/36e7ea23a010ff91053f82ac4f6a9bfc698503f9/remoteid/canonical.yaml#L11), "`/v1/dss/...`".  So, the RID endpoints (for instance) cannot be at `/dss/rid/v1` because they must be at `*/v1/dss/...`.  We could have `https://example.com/dss/rid/v1/dss/identification_service_areas`, and then the base RID DSS URL would be `https://example.com/dss/rid`.  But, it seems like that first `dss` is unnecessary.  Currently, we have `https://example.com/v1/dss/identification_service_areas` where the base RID DSS URL is `https://example.com`.  But the issue with having both `rid` and `scd` APIs' base URL being the same (`https://example.com`) is that if they happened to have identically-named endpoints (e.g., /subscriptions, if the two APIs didn't require the /v1/dss or /dss/v1 prefixes), our current approach wouldn't work.  To be robust to adding arbitrary APIs, it seems like we should have a prefix per API (specifically, one for rid, one for scd, and one for aux).

Reply:
I've added a commit per our conversation that makes the `aux` subsystem more explicitly distinct from `rid` and `scd`.  For future reference, my expanded explanation was:

The RID API includes, e.g., /v1/dss/subscriptions.  So, to implement the API, we have to have an endpoint that ends with that subpath.  The SCD API includes, e.g., /dss/v1/subscriptions.  So, to implement the API, we have to have an ednpoint that ends with that subpath.  Currently, the base URL we're using to support both APIs is just https://whatever-our-hostname-is.com.  That's not robust because two APIs may easily specify the same endpoint.  For instance, if the SCD API were more consistent with the RID API (which the standards group probably will do), we would need to implement */v1/dss/subscriptions for RID and we would need to implement */v1/dss/subscriptions for SCD.  Currently, that's impossible with a single common base URL like we have now.  So, I think we'll need to add a prefix per API.  So instead of https://example.com/v1/dss/subscriptions, the RID Subscriptions endpoint would be found at https://example.com/rid/v1/dss/subscriptions.  In accordance with that, I think all aux stuff should be at https://example.com/aux/*  I'm fine with versioning in the URL, so probably the version URL should be https://example.com/aux/v1/version  There's no need for /dss in that path; it's only in the RID & SCD paths because those APIs (which we don't control as InterUSS Project members) say it needs to be.  Since we can name aux endpoints whatever we want, there's no reason for us to put /dss in.  Given that we want the version endpoint to be https://example.com/aux/v1/version eventually, I think we can either specify /aux/v1/version in the proto (achieving our goal immediately) or specify /v1/version in the proto, with the expectation that we'll add the aux prefix in the future (in the same way we'll add the rid and scd prefixes in the future).